### PR TITLE
Add ability to configure batch policy

### PR DESCRIPTION
**Pull request by jekanik**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -26,6 +26,8 @@
         <java.version>1.8</java.version>
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
+        <maven.javadoc.plugin.version>3.2.0</maven.javadoc.plugin.version>
```
Comment (Aloren) [inline pom.xml:29]: Both versions are provided by parent. Please use the ones from the parent and do not override them

File: spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/aerospike/AerospikeProperties.java [hunk 2]
```diff
@@ -94,15 +94,17 @@
...
 
     private WritePolicyDefault write = new WritePolicyDefault();
 
+    private BatchPolicyDefault batch = new BatchPolicyDefault();
+
     /**
      * For more details on each option see corresponding field documentation in {@link com.aerospike.client.policy.Policy}.
      */
     @Data
-    public static class ReadPolicyDefault {
```
Comment (Aloren) [inline spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/aerospike/AerospikeProperties.java:105]: Please leave ReadPolicyDefault class for configuring read, as using PolicyDefault for read-configuration is not readable and will not give an ability to add properties in the future without breaking someone who is using AerospikeProperties directly.

**Review by Aloren — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (jekanik) [reply pom.xml]: Fixed

File: spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/aerospike/AerospikeProperties.java (hunk 2 above)
Comment (jekanik) [reply spring-boot-autoconfigure-data-aerospike/src/main/java/org/springframework/boot/autoconfigure/aerospike/AerospikeProperties.java:105]: Created separate ReadPolicyDefault for read config.

**Pull request merged**